secret-manager/README.md

```diff
@@ -0,0 +1,55 @@
+# Quarkiverse - Google Cloud Services - Secret Manager
+
+This extension allows to inject the Google Cloud Platform Secret Manager client library inside your Quarkus application.
+
+## Bootstrapping the project
+
+First, we need a new project. Create a new project with the following command (replace the version placeholders with the correct ones):
+
+```shell script
+mvn io.quarkus:quarkus-maven-plugin:${quarkusVersion}:create \
+    -DprojectGroupId=org.acme \
+    -DprojectArtifactId=secretmanager-quickstart \
+    -Dextensions="resteasy-jackson,io.quarkiverse.googlecloudservices:quarkus-google-cloud-secret-manager:${googleCloudServicesVersion}"
+cd secretmanager-quickstart
+```
+
+This command generates a Maven project, importing the Google Cloud Secret Manager extension.
+
+If you already have your Quarkus project configured, you can add the `quarkus-google-cloud-secret-manager` extension to your project by running the following command in your project base directory:
+```shell script
+./mvnw quarkus:add-extension -Dextensions="io.quarkiverse.googlecloudservices:quarkus-google-cloud-secret-manager:${googleCloudServicesVersion}"
+```
+
+This will add the following to your pom.xml:
+
+```xml
+<dependency>
+    <groupId>io.quarkiverse.googlecloudservices</groupId>
+    <artifactId>quarkus-google-cloud-secret-manager</artifactId>
+    <version>${googleCloudServicesVersion}</version>
+</dependency>
+```
+
+## Some example
```

Comment:
Please provides a Java example as all other example are in Java.

Reply:
Indeed, it's nice to be consistent. However, I think it's nice that the project has an example or two in Kotlin as well, for those (me included) who are using Kotlin with Quarkus native, just to prove that this project also can be helpful in that context. I added both a Java and a Kotlin version in the newly pushed commit, perhaps that's a suitable solution?

Reply:
The issue here is consistency as this will be the only guide that incldues Kotlin example.
So, if you plan to contribute Kotlin example for the other guide, I'm OK with it (I'll love to see more Kotlin with Quarkus), otherwise it is better to avoid it.

The readability also is a concern, when using mutiple example on the same thing on different languages I expect the design to only show one language and having way to toggle from one to the other, we didn't have this currently ...

Reply:
Yeah, I see the concerns you're pointing out. I guess most people using Kotlin these days also know Java pretty well, and are able to convert a Java example into Kotlin - and if not, IntelliJ does that pretty good as well.